src/Dev/Install/InstallRequirements.php

```diff
@@ -259,6 +259,8 @@ public function check()
 
         $this->requireWriteable('index.php', array("File permissions", "Is the index.php file writeable?", null));
 
+        $this->requireWriteable('.env', ["File permissions", "Is the .env file writeable?"], false, false);
```

Comment:
missing `null` for third param, other than that it works fine :)

Reply:
Oops right. Fixed.